### PR TITLE
feat: add container for gcloud workstation, INPRO-1640

### PR DESCRIPTION
**Pull request by dusanscensny**

### CODE REVIEW

File: gcloud-workstation-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# As checked here https://console.cloud.google.com/artifacts/docker/cloud-workstations-images/us-central1/predefined/code-oss
```
Comment (tyriis) [inline gcloud-workstation-base/Dockerfile:1]: Are you sure you are using pre-commit? End of file

File: gcloud-workstation-base/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+## Notice
+
+<!-- MARKDOWN LINKS & IMAGES -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+<!-- Links -->
+
+[pre-commit]: https://pre-commit.com/
```
Comment (tyriis) [inline gcloud-workstation-base/README.md:50]: Most of these lines does not make sense

File: gcloud-workstation-base/Dockerfile (hunk 1 above)
Comment (dusanscensny) [reply gcloud-workstation-base/Dockerfile:1]: Will check on Monday. Committed first time from remote machine

File: gcloud-workstation-base/README.md (hunk 2 above)
Comment (dusanscensny) [reply gcloud-workstation-base/README.md:50]: Did you mean links? Will clean up them. I tried to use template readme, but wrong copy paste

File: gcloud-workstation-base/Dockerfile (hunk 1 above)
Comment (dusanscensny) [reply gcloud-workstation-base/Dockerfile]: was not executed. Now should be ok

File: gcloud-workstation-base/README.md (hunk 2 above)
Comment (dusanscensny) [reply gcloud-workstation-base/README.md]: Check now please. Thanks

**Comment by tyriis** [comment]
still taskfile and pre-commit are not used here, maybe on top level, would remove it from readme

**Comment by dusanscensny** [comment]
> still taskfile and pre-commit are not used here, maybe on top level, would remove it from readme

Ok. Will remove them from subfolder readme

**Review by tyriis — APPROVED** [review]

File: gcloud-workstation-base/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# there is no versionning so I will keep it on latest version
+FROM us-central1-docker.pkg.dev/cloud-workstations-images/predefined/code-oss:latest
+
+LABEL name=@strg-at/postgres-db-init
```
Comment (jazzlyn) [inline gcloud-workstation-base/Dockerfile:5]: wrong label :) @dusanscensny

Comment (dusanscensny) [reply gcloud-workstation-base/Dockerfile]: Thanks updated

**Review by jazzlyn — APPROVED** [review]

**Pull request merged**